bukuserver/response.py

```diff
@@ -1,20 +1,20 @@
 from typing import Any, Dict
 from enum import Enum
+from http import HTTPStatus
 from flask import jsonify
-from flask_api.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_404_NOT_FOUND, HTTP_410_GONE
```

Comment:
These constants appear to be our primary purpose of pulling in Flask-API as a dependency :sweat_smile: